sci-mathematics/minisat/minisat-2.2.0-r4.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Small yet efficient SAT solver with reference paper"
+HOMEPAGE="http://minisat.se/Main.html"
+SRC_URI="http://minisat.se/downloads/${P}.tar.gz
+	doc? ( http://minisat.se/downloads/MiniSat.pdf )"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+LICENSE="MIT"
+
+IUSE="debug doc extended-solver"
+
+DEPEND="sys-libs/zlib"
+RDEPEND="${DEPEND}"
+
+DOCS=( README doc/ReleaseNotes-2.2.0.txt )
+
+S=${WORKDIR}/${PN}
+
+src_prepare() {
+	default
+	sed -e "s/\$(CXX) \$^/\$(CXX) \$(LDFLAGS) \$^/" \
+		-i -e "s|-O3|${CFLAGS}|" mtl/template.mk || die
+}
+
+src_configure() {
+	myconf=$(usex debug d r)
+	myext=$(usex debug debug release)
+	mydir=$(usex extended-solver simp core)
+
+	tc-export CXX
+
+	if has_version "=sci-mathematics/minisat-2.1*" ; then
+		elog ""
+		elog "The minisat2 2.1 and 2.2 ABIs are not compatible and there"
+		elog "is currently no slotting.  Please mask it yourself (eg, in"
+		elog "packages.mask) if you still need the older version."
+		elog ""
+		epause 5
+	fi
```

Comment:
so we want to keep the 2.1 range? Are there any revdeps for the 2.1 version in the tree?